README.md

```diff
@@ -8,41 +8,54 @@ The technical background and architecture of Ship-it can be found in the overvie
 
 ## Local Development
 
-This project uses skaffold for local development and testing. To get started,
-you'll need to install these tools locally:
+This project uses kind for local development and testing. To get started,
+you'll need to install these tools to your development machine:
 
 * helm: https://github.com/helm/helm
-* minikube: https://github.com/kubernetes/minikube
-* skaffold: https://github.com/GoogleContainerTools/skaffold
+* kind: https://github.com/kubernetes-sigs/kind
 
-1. Start minikube cluster
+1. Create a new kind cluster
 
 ```bash
-$ minikube start
+$ make kind-up
 ```
 
 2. Update kubectl's cluster context
 
 ```bash
-$ minikube update-context
+$ export KUBECONFIG=$(kind get kubeconfig-path --name="ship-it-dev")
 $ kubectl config current-context
-minikube
+kubernetes-admin@ship-it-dev
 ```
 
-3. Install tiller in minikube
+3. Build local images
```

Comment:
It would be great to automate this instead of having it as a manual step

Reply:
This step would be a bit nicer if there was a single `make images` (or whatever) target instead of 3 different ones. The env target thing was neat but it's starting to annoy me in practice. Might try to get it down to a single target, even if there's more duplication in the makefile.